js/PulsarFormComponent.js

```diff
@@ -83,10 +83,31 @@ PulsarFormComponent.prototype.initTimePickers = function () {
  * Initiate a date picker on data-datepicker fields using pickaday
  */
 PulsarFormComponent.prototype.initDatePickers = function () {
-    // Attach basic pikaday to datepicker fields
-    this.$html
-        .find('[data-datepicker=true]')
-        .pikaday({ format: 'DD/MM/YYYY' });
+    const datepickers = this.$html.find('[data-datepicker="true"]');
+    let defaultDateFormat = 'DD/MM/YYYY';
+
+    datepickers.each((index, element) => {
+        const dateFormat = element.getAttribute('data-format');
+
+        // Add missing data-format to hardcoded examples
+        if (!dateFormat) {
+            element.setAttribute('data-format', 'default');
```

Comment:
Why do we need to add this attribute? I was under the impression that this component uses it to determine the format, so if the attribute is missing, can't we just use the default format and not add the attribute?

Reply:
How does that make this backwards compatible? I can't see anything in this service to suggest that the attribute value of `data-format` is used again.